tools/dev-doctor/rootcmd.go

```diff
@@ -147,6 +147,28 @@ func rootCmdRun(cmd *cobra.Command, args []string) {
 			versionRegexp: regexp.MustCompile(`pip (\d+\.\d+\S*)`),
 		},
 		dockerGroupCheck{},
+		&binaryCheck{
+			name:          "golangci-lint",
+			ifNotFound:    checkWarning,
+			versionArgs:   []string{"version"},
+			versionRegexp: regexp.MustCompile(`(\d+\.\d+\S*)`),
+			minVersion:    &semver.Version{Major: 1, Minor: 27, Patch: 0},
+			hint:          "See https://golangci-lint.run/usage/install/#local-installation.",
+		},
+		&binaryCheck{
+			name:          "cfssl",
```

Comment:
Should the checks for `cfssl` and `cfssljson` be moved to the list of `binaryCheck`s above, or is there a particular reason to have them below the `dockerGroupCheck{}`?

Reply:
good point, let me move them up. Thanks